dbms/src/Interpreters/TextLog.cpp

```diff
@@ -0,0 +1,77 @@
+#include <Interpreters/TextLog.h>
+#include <DataTypes/DataTypeEnum.h>
+#include <DataTypes/DataTypesNumber.h>
+#include <DataTypes/DataTypeDateTime.h>
+#include <DataTypes/DataTypeDate.h>
+#include <DataTypes/DataTypeString.h>
+#include <Common/ClickHouseRevision.h>
+#include <array>
+
+namespace DB
+{
+
+template <> struct NearestFieldTypeImpl<Message::Priority> { using Type = UInt64; };
+
+Block TextLogElement::createBlock()
+{
+    auto priority_datatype = std::make_shared<DataTypeEnum8>(
+        DataTypeEnum8::Values
+        {
+                {"FATAL",          static_cast<Int8>(Message::PRIO_FATAL)},
```

Comment:
I prefer `Fatal` not `FATAL`, like in text log itself.

Reply:
Fixed